src/onboarding/restore-backup/onboarding-restore-backup-empty-cloud.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, html, css, type CSSResultGroup } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { LocalizeFunc } from "../../common/translations/localize";
+import "../../components/ha-button";
+import "../../components/ha-icon-button-arrow-prev";
+import { fireEvent } from "../../common/dom/fire_event";
+import { navigate } from "../../common/navigate";
+import { removeSearchParam } from "../../common/url/search-params";
+import { onBoardingStyles } from "../styles";
+
+@customElement("onboarding-restore-backup-empty-cloud")
```

Comment:
Can we change the name for `empty-cloud` to `no-cloud-backup`?

```suggestion
@customElement("onboarding-restore-backup-no-cloud-backup")
```